packit/copr_helper.py

```diff
@@ -66,6 +66,41 @@ def get_copr_settings_url(
 
         return f"{copr_url}/coprs/{owner}/{project}/{section}/"
 
+    def _update_chroot_specific_configuration(
+        self,
+        project: str,
+        owner: Optional[str] = None,
+        targets_dict: Optional[Dict] = None,  # chroot specific configuration
+    ):
+        """
+        Using the provided targets_dict, update chroot specific configuration
+        """
+        if targets_dict:
+            # let's update chroot specific configuration
+            for chroot_name, chroot_configuration in targets_dict.items():
+                if set(chroot_configuration.keys()).intersection(
+                    CHROOT_SPECIFIC_COPR_CONFIGURATION
+                ):
+                    # only update when needed
+                    copr_chroot_configuration = (
+                        self.copr_client.project_chroot_proxy.get(
+                            ownername=owner, projectname=project, chrootname=chroot_name
+                        )
+                    )
+                    update_dict = {}
+                    for c in CHROOT_SPECIFIC_COPR_CONFIGURATION:
+                        if copr_chroot_configuration.get(
+                            c, None
+                        ) != chroot_configuration.get(c, None):
```

Comment:
Just wondering, does it look better to you with the explicit [default](https://docs.python.org/3/library/stdtypes.html#dict.get)?

Reply:
good point

actually, the default is wrong and I forgot to update it - all three have default `[]` in Copr so we should change it to that instead of `None` to not trigger useless calls in the future